helmfile.d/values/openstack-monitoring.yaml.gotmpl

```
@@ -1,2 +1,12 @@
 openstackMonitoring:
     enabled: {{ .Values.openstackMonitoring.enabled }}
+serviceMonitor:
+  {{- if .Values | get "clusterApi.enabled" false }}
+  labels:
+    app: openstack-cloud-controller-manager
+  namespace: openstack-system
+  {{- else }}
+  labels:
+    k8s-app: openstack-cloud-controller-manager
+  namespace: kube-system
+  {{- end }}
```

Suggestion: Can be simplified with:

```suggestion
serviceMonitor:
  labels:
    {{- if .Values | get "clusterApi.enabled" false }}
    app: openstack-cloud-controller-manager
    {{- else }}
    k8s-app: openstack-cloud-controller-manager
    {{- end }}
  namespace: {{ .Release.Namespace }}
```

---

fixed